### ISSUE TITLE
Flight_pilot + Camera

### ISSUE THREAD

**Issue by sbhatti915** [uzh-rpg/flightmare#117]
I am having trouble integrating the flight_pilot.cpp and camera.cpp to have Flightmare move with our controller and to see the images produced from the camera. Has anyone been able to successfully integrate these 2?

**Comment by JosephFeld** [comment]
I got this working! Right now it's in my private repo - if you still want this dm me.

**Comment by hanif-hazemi** [comment]
Hi! @JosephFeld. Would you mind sharing how you do this? I copy the nodes (camera and rviz) from the camera.launch file to rotors_gazebo.launch but no luck so far (flight_pilot_node crashes). Appreciate your help.. @yun-long

**Comment by sirengarcia** [comment]
> I got this working! Right now it's in my private repo - if you still want this dm me.

Sorry to bother u @JosephFeld, I've been struggling with the implementation of this feature for a month now, would you mind sharing your method please?

**Comment by 8ajpai** [comment]
Hey @sirengarcia @hanif-hazemi did you figure it out? @JosephFeld How do i dm you?

**Comment by sirengarcia** [comment]
> Hey @sirengarcia @hanif-hazemi did you figure it out? @JosephFeld How do i dm you?

yes, I have solved it. I used the rl modules. Do u need help still?

**Comment by sirengarcia** [comment]
yes, I have solved it. I used the rl modules. Do u need help still? @8ajpai

**Comment by 8ajpai** [comment]
Hey @sirengarcia that would be helpful. Can we talk over email? s7utbajp@uni-bonn.de

**Comment by cmakelabs** [comment]
Hi @sirengarcia @8ajpai  I know this is not the right place to discuss, but I opened a discussion  [here](https://github.com/uzh-rpg/flightmare/discussions/204) but no one has answered me, I would be really grateful if you can answer me, on how to move the camera of the scene, like in gazebo using mouse

**Comment by jason8372** [comment]
Facing the same issue. Can someone help me on integrating camera feature on flight_pilot.cpp(getting RGB image in ROS topic)?
